Gemfile

```diff
@@ -37,6 +37,7 @@ gem "slack-notifier", "~> 2.3", ">= 2.3.2"
 gem "sprockets", "~> 3.7", ">= 3.7.2"
 gem 'next_rails'
 gem "json-canonicalization", '0.3.1'
+gem 'msgpack', "~> 1.4.4"
```

Comment:
@ashwinisukale do we need this?

Reply:
good question :D , I guess I locked the version number as this is a dependency gem, I dont remember what was the error, let me confirm the same.

Reply:
I got the same error for Volpino repo, hence sharing that here too, Bundle install fails if you dont lock the msg pack version. As this is dependency gem

```

An error occurred while installing msgpack (1.4.2), and Bundler cannot continue.

In Gemfile:
  bootsnap was resolved to 1.9.1, which depends on
    msgpack
    ```